samples/integration/vcpkg-all-smoke/src/main.cpp

```diff
@@ -67,7 +67,7 @@ int main()
 
     std::cout << "Successfully Created the Clients";
   }
-  catch (std::exception exception)
+  catch (std::exception const& exception)
   {
     std::cout << "Exception: " << exception.what();
```

Comment:
exit(1) when the ctor throws, that will cause the smoke test to fail.

Reply:
If we want to use another exit code from the program, here I'd rather use `return 1`, not `exit(1)`.

But I think it draws attention out from the sample code. Instead, I'd remove `return 0` below (in C++ `int main()` is the only function that is allowed to have no `return` statement).